README.md

```diff
@@ -16,28 +16,49 @@ In order to use Fixme all you need to do is install it:
 
 > npm install -g fixme
 
-*Note: There really shouldn't be much reason to globally install it...*
+###CLI Usage:
 
-Require it:
+Typing 
+
+> fixme
+
+in a directroy will search all `.js` files in all subdirectories except some preconfigured once _(vendors/**, vendor/**, bower_components/**, jspm_packages/**, node_modules/**)_
+
+Note: Passing custom options from the command line is currently not supported.
+
+### Programmatic Usage
+
+When using it **programtically**
+
+_like so:_
 
 ```javascript
 var fixme = require('fixme');
 ```
 
-And finally; configure it when you call it:
+the follwing options can be configured:
 
 ```javascript
 // All values below are Fixme default values unless otherwise overridden here.
-fixme({
+var options = {
   path:                 process.cwd(),
   ignored_directories:  ['node_modules/**', '.git/**', '.hg/**'],
   file_patterns:        ['**/*.js', 'Makefile', '**/*.sh'],
   file_encoding:        'utf8',
-  line_length_limit:    1000
+  line_length_limit:    1000,
+  color:                true
+};
+```
+
+when called like so:
+
+```javascript
+fixme(options, function(outptu){ 
```

Comment:
The variable passed into the function here is "outptu" (I believe you meant "output")